glossary.md

```diff
@@ -140,10 +158,20 @@ A genre within AVNs that focuses on building emotional connections and romantic
 
 A method of naming MC's sibling to bypass the Patreon incest restrictions. Some games will let you change the title of this roommate to retain immersion, while other games will directly search and replace "sister" for "roommate". As a result, many games will reference how taboo it is to 'have sex with your roommate', confusing those not in the know. See [Landlady](#landlady) for mother/child.
 
+## Sandbox
+
+Games that center their gameplay on daytime cycles or a world map that the player can access to engage in a world event. There are story arcs that are only accessible at different points in the story, and/or different points in the day. These games are less linear than regular AVNs and give the players the freedom to explore the game world at their own pace.
+
+Games can have sandbox elements like free-roam events without being considered sandbox games as long as the gameplay remains mostly linear.
+
 ## Scripting
 
 The process of writing, coding, or implementing the script and dialogue for a visual novel, including branching narratives, dialogue trees, and event triggers.
 
+## Side Girl (SG)
+
+[Love Interests](#love-interest-li)) that offer side content during the story, but that don’t affect the development of the plot in any major or vital way.
```

Comment:
- Extra ) in this line
- Comma before but is not necessary
- "that" after but is unnecessary